tests/req.js

```diff
@@ -1,32 +1,35 @@
+'use strict';
+
 var test = require('tape');
 var MockRequest = require('../index.js').Request;
-var MockResponse = require('../index.js').Response;
 var Cookies = require('cookies');
-var PassThrough = require('stream').PassThrough;
+var PassThrough = require('readable-stream').PassThrough;
+
+test('MockRequest: Create request with cookie', function (t) {
+  t.plan(3);
 
-test('Create request with cookie', function(t) {
   var req = new MockRequest({
     url: '/hello',
     cookies: {
       foo: 'bar',
       bar: 'baz'
     }
   });
-  var res = new MockResponse();
```

Comment:
passing a res to Cookies was un-necessary since this test doesn't interact with response. I removed it to keep the tests as small and tight as possible.